test/integration/model-name/stanc.expected

```diff
@@ -1,16 +1,4 @@
   $ ../../../../install/default/bin/stanc --name=foo  bad.stan
 
-Semantic error in 'bad.stan', line 3, column 6 to column 9:
-   -------------------------------------------------
-     1:  data {
-     2:    // declare matching model name should fail
-     3:    int foo;  
-               ^
-     4:  }
-     5:  parameters {
-   -------------------------------------------------
-
-Identifier 'foo' clashes with model name.
-
```

Comment:
Where did this error go? It's still supposed to fail, right?

Reply:
Yeah, this is weird. Will check.